.tav.yml

```diff
@@ -258,7 +258,7 @@ apollo-server-express-2_15:
   # but only the latest MAJOR.MINOR.x to reduce the test matrix.
```

Comment:
Perhaps update the "We want this version range ..." comment just above this line?